ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env

```diff
@@ -1 +1,2 @@
-discussion_forum_token= {{ discussion_forum_token }}
\ No newline at end of file
+nodebb_service_url={{ discussionsmw_nodebb_service_url | default('http://nodebb-service:4567') }}
+authorization_token={{ discussionsmw_nodebb_authorization_token }}
```

Comment:
please add a comment with pointer for creating the auth token.

Reply:
1. Login to  Nodebb site as a Admin.
2.  go to admin panel.
3. click on Plugins => Write API 
4. enter UID as 1 and click create token (Auth token).